src/randomeffectsterm.jl

```diff
@@ -6,6 +6,10 @@ end
 Base.show(io::IO, t::RandomEffectsTerm) = print(io, "($(t.lhs) | $(t.rhs))")
 StatsModels.is_matrix_term(::Type{RandomEffectsTerm}) = false
 
+function StatsModels.termvars(t::RandomEffectsTerm)
+    vcat(StatsModels.termvars(t.lhs), StatsModels.termvars(t.rhs))
```

Comment:
I was going to say you might want to use `union` (like StatsModels does) but there shouldn't be any duplication between the lhs and rhs so there's really no need.

Reply:
Yeah, duplication on the lhs and rhs would be very bad.

Reply:
may be worth checking that in the constructor actually...